App.tsx

```diff
@@ -0,0 +1,34 @@
+import 'react-native-gesture-handler';
+import React from 'react';
+import {ThemeProvider} from 'styled-components/native';
+import theme from './src/themes';
+import Header from './src/components/header';
+import crypto from './src/screens/cryptos/index';
```

Comment:
Components should have the name in PascalCase

Reply:
If there is an index file in the component's folder you can simply use:  
import crypto from './src/screens/cryptos

Also Rename the folder of the component: Cryptos. in PascalCase